main/EDParamsMod.F90

```diff
@@ -2,14 +2,14 @@ module EDParamsMod
    !
    ! module that deals with reading the ED parameter file
    !
-  
+
+   use FatesConstantsMod, only : r8 => fates_r8
    use FatesParametersInterface, only : param_string_length
    use FatesGlobals        , only : fates_log
    use FatesGlobals        , only : endrun => fates_endrun
 
    ! CIME Globals
    use shr_log_mod         , only : errMsg => shr_log_errMsg
-   use shr_kind_mod        , only: r8 => shr_kind_r8
```

Comment:
What predicated the change to `fates_r8`?  Or put another way, what is the history behind the use of `shr_kind_mod`?

Reply:
The unit test pre-processor code will actually define r8's slightly differently than how we normally do for a standard elm/clm-fates run, in that it will define it based on the iso c standard library.  Its important to use that library for the unit tests.  So in order to use the right definition, we must point to fates constants (where it is defined).  I don't think there is much (at all?) difference between our definition of an 8-byte real, and the definition in shr_kind_mod.  But its more consistent at the very least.